### PR TITLE
[docs] updates for TPC-C page

### LINKED ISSUE #9745: [Docs] update TPC-C benchmark section
Update the [Benchmark > TPC-C section](https://docs.yugabyte.com/latest/benchmark/tpcc-ysql/) with the information in [this Google doc](https://docs.google.com/spreadsheets/d/1gPPrML-R0CvDr-AVeK-tx3gMlDaGczpwIpgBGnMs4LA/edit#gid=1608563644).

In particular, the _Optimizations_, _Published Results_, and _Cluster Configuration_ tabs are what we're looking for. If we need more information on the results, use the _Phase 3_ tab.

ALSO...take a careful read over each of the tabs in step 3 ("3. Run the TPC-C benchmark") to make sure the machine types are all up to date. The 10,000 tab has a mismatch, so check that and all the others.

(@ameyb and @dcomingore will be reviewers.)

### PR DESCRIPTION
**Pull request by dcomingore**
@ameyb @polarweasel @hbhanawat 

Pull Request for TPCC Doc Page Enhancements:

Fixes #9745 

Modified _index.md in latest:
1. Added Microsoft Azure Test Results
2. Added Performance Team TPCC Best Practices

@netlify /latest/benchmark/tpcc-ysql/

### CODE REVIEW

File: docs/content/latest/architecture/design-goals.md [hunk 1]
```diff
@@ -20,10 +20,12 @@ This page outlines the design goals with which YugabyteDB has been built.
 
 YugabyteDB offers strong consistency guarantees in the face of a variety of failures. It supports distributed transactions.
 
-### CAP theorem
+### CAP Theorem & Split-Brain
```
Comment (polarweasel) [inline docs/content/latest/architecture/design-goals.md:23]: Sentence case for headings

```suggestion
### CAP theorem and split-brain
```

File: docs/content/latest/architecture/design-goals.md [hunk 2]
```diff
@@ -20,10 +20,12 @@ This page outlines the design goals with which YugabyteDB has been built.
...
 
+Split-brain is a computing scenario that represents data and availability inconsistencies which arise when a distributed system incurs a network partition. For YugabyteDB, when a network partition occurs, the remaining ( majority for write acknowledgement purposes ) RAFT group peers elect a new tablet leader. YugabyteDB makes use of a concept known as leader leases which ensures a single tablet leader exists throughout the entire distribured system including when network partitions occur. Leader leases have a default value of two seconds and can be configured to use a different value. Because of YugabyteDB's architecture, the distributed database is not suspectible to the split-brain condition.
```
Comment (polarweasel) [inline docs/content/latest/architecture/design-goals.md:27]:
```suggestion
Split-brain is a computing scenario in which data and availability inconsistencies arise when a distributed system incurs a network partition. For YugabyteDB, when a network partition occurs, the remaining (majority for write acknowledgement purposes) RAFT group peers elect a new tablet leader. YugabyteDB implements _leader leases_, which ensures that a single tablet leader exists throughout the entire distributed system including when network partitions occur. Leader leases have a default value of two seconds, and can be configured to use a different value. This architecture ensures that YugabyteDB's distributed database is not susceptible to the split-brain condition.
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
Small edits...

**Comment by polarweasel** [comment]
@dcomingore you seem to have clobbered your TPC-C edits with split-brain edits for the architecture section. Can you add the TPC-C edits back in here, and we'll consider this PR for both?

File: docs/content/latest/architecture/design-goals.md [hunk 3]
```diff
@@ -20,10 +20,20 @@ This page outlines the design goals with which YugabyteDB has been built.
 
 YugabyteDB offers strong consistency guarantees in the face of a variety of failures. It supports distributed transactions.
 
-### CAP theorem
+<<<<<<< Updated upstream
```
Comment (polarweasel) [inline docs/content/latest/architecture/design-goals.md:23]: leftover merge artifact... (you do need the blank line, though!)
```suggestion

```

File: docs/content/latest/architecture/design-goals.md [hunk 4]
```diff
@@ -20,10 +20,20 @@ This page outlines the design goals with which YugabyteDB has been built.
...
+
+Split-brain is a computing scenario in which data and availability inconsistencies arise when a distributed system incurs a network partition. For YugabyteDB, when a network partition occurs, the remaining (majority for write acknowledgement purposes) RAFT group peers elect a new tablet leader. YugabyteDB implements _leader leases_, which ensures that a single tablet leader exists throughout the entire distributed system including when network partitions occur. Leader leases have a default value of two seconds, and can be configured to use a different value. This architecture ensures that YugabyteDB's distributed database is not susceptible to the split-brain condition.
+>>>>>>> Stashed changes
```
Comment (polarweasel) [inline docs/content/latest/architecture/design-goals.md:35]: Delete the whole range, leaving just the blank line.
```suggestion

```

File: docs/content/latest/benchmark/tpcc-ysql/_index.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ isTocNested: true
 ## Overview
 Follow the steps below to run the [TPC-C workload](https://github.com/yugabyte/tpcc) against YugabyteDB YSQL. [TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types and complexity that are either either executed online or queued for deferred execution.
 
-### Results at a glance
+### Results at a glance ( Amazon Web Services )
```
Comment (polarweasel) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:23]: No blanks in the parens
```suggestion
### Results at a glance (Amazon Web Services)
```

File: docs/content/latest/benchmark/tpcc-ysql/_index.md [hunk 6]
```diff
@@ -30,6 +30,16 @@ Follow the steps below to run the [TPC-C workload](https://github.com/yugabyte/t
 
 All the nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for `30 minutes` after the loading of the data.
```
Comment (polarweasel) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:31]: I know this paragraph isn't yours, but there's no need for the code font for "30 minutes" :)
```suggestion
All the nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for 30 minutes after the loading of the data.
```

File: docs/content/latest/benchmark/tpcc-ysql/_index.md [hunk 7]
```diff
@@ -63,7 +73,18 @@ Other options like username, password, port, etc. can be changed using the confi
 <password></password>
 ```
 
-## 3. Run the TPC-C benchmark
+## 3. Best Practices
```
Comment (polarweasel) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:76]: Sentence case headings
```suggestion
## 3. Best practices
```

File: docs/content/latest/benchmark/tpcc-ysql/_index.md [hunk 8]
```diff
@@ -63,7 +73,18 @@ Other options like username, password, port, etc. can be changed using the confi
...
+
+<b>Latest TPCC Code</b><br/>
+Use the latest enhancements of the Yugabyte TPCC application `git clone`.
+
+<b>Pre-Compacting Tables</b><br/>
+Pre-compact tables using the compact_table command in the yb-admin utility. 
+
+<b>Warming the Database</b><br/>
+Make use of the --warmup-time-secs flag when calling the execute phase of the TPCC benchmark.
```
Comment (polarweasel) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:85]: Run-in headings are more appropriate here. Also, what do you mean in the "latest tpcc code" item?
```suggestion
**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application `git clone`.

**Pre-compacting tables:** Pre-compact tables with the yb-admin utility's `compact_table` command.

**Warming the database:** Use the `--warmup-time-secs` flag when you call the execute phase of the TPCC benchmark.
```

File: docs/content/latest/benchmark/tpcc-ysql/_index.md [hunk 9]
```diff
@@ -30,6 +30,16 @@ Follow the steps below to run the [TPC-C workload](https://github.com/yugabyte/t
...
 
+### Results at a glance ( Microsoft Azure )
+| Warehouses| TPMC | Efficiency (approx) | Cluster Details
+-------------|-----------|------------|------------|
+50    | 639.4 | 99.44%   | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks
+100   | 1,271.37 | 98.86%   | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks
+1000  | 12,523.97 | 97.39%   | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks
+2000  | 25,407.43 | 98.78%   | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks
+
+All the nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for `30 minutes` after the loading of the data.
```
Comment (polarweasel) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:41]: General cleanup, including better table markup...
```suggestion
### Results at a glance (Microsoft Azure)

| Warehouses | TPMC | Efficiency (approx) | Cluster details |
| :--------- | :--- | :-----------------: | :-------------- |
| 50 | 639.4 | 99.44% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
| 100 | 1,271.37 | 98.86% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
| 1000 | 12,523.97 | 97.39% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
| 2000 | 25,407.43 | 98.78% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |

All nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster, and was deployed in the same zone as the DB cluster. Each test was run for 30 minutes after the loading of the data.
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
More changes... Use the "add suggestion to batch" buttons and you'll only need to do one more commit :)

File: docs/content/latest/benchmark/tpcc-ysql/_index.md (hunk 8 above)
Comment (dcomingore) [reply docs/content/latest/benchmark/tpcc-ysql/_index.md:85]: As opposed to downloading a specific release of the tpcc application - use git clone and get all code updates

**Comment by dcomingore** [comment]
Used the 'add suggestion' GH feature - are we good? Thanks! @polarweasel

File: docs/content/latest/benchmark/tpcc-ysql/_index.md (hunk 8 above)
Comment (polarweasel) [reply docs/content/latest/benchmark/tpcc-ysql/_index.md]: Ah ok, I see what you mean. I'll make a quick update and I think we'll be good to go.

**Review by polarweasel — APPROVED** [review]
All looks good now @dcomingore!

File: docs/content/latest/benchmark/tpcc-ysql/_index.md [hunk 10]
```diff
@@ -63,7 +74,15 @@ Other options like username, password, port, etc. can be changed using the confi
 <password></password>
 ```
 
-## 3. Run the TPC-C benchmark
+## 3. Best practices
+
+**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application `git clone`.
```
Comment (hbhanawat) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:79]: This is something we can avoid now. We are planning a release in this week. Once that is done, one can download the latest binaries from the release page.

**Comment by hbhanawat** [comment]
Also in "Get TPC-C binaries", we will have to change the link to point to the latest release. I will share the link by the end of this week.

File: docs/content/latest/benchmark/tpcc-ysql/_index.md (hunk 10 above)
Comment (dcomingore) [reply docs/content/latest/benchmark/tpcc-ysql/_index.md:79]: Thanks @hbhanawat! Going forward and similar to the past reason for this initially, the latest enhancements will always be found in the source itself. Recommend we keep git clone unless there is a reason not to?

Comment (polarweasel) [inline docs/content/latest/benchmark/tpcc-ysql/_index.md:79]: Is this a useful change? It's at least two complete sentences, which seems like an improvement ;-)
```suggestion
**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest released version, or, to get the very latest changes, you can clone the repository and build from source.
```

**Comment by dcomingore** [comment]
Hey Team - can I "Squash and merge" today? @hbhanawat @polarweasel @hbhanawat - I think your team is going to do another round on the tpcc docs after this PR as well

**Comment by polarweasel** [comment]
> Hey Team - can I "Squash and merge" today? @hbhanawat @polarweasel @hbhanawat - I think your team is going to do another round on the tpcc docs after this PR as well

@dcomingore I'm fine with merging this, provided Hemant is happy with it. (Or, if his team's doing some updates shortly anyway, we could just merge it and they can correct anything they'd like to...)

**Comment by dcomingore** [comment]
Copy that

**Comment by hbhanawat** [comment]
@dcomingore This is fine. I will raise another PR for the link changes.

**Pull request merged**